USFMToolsSharp.Renderers.HTML/HtmlRenderer.cs

```diff
@@ -67,10 +67,22 @@ public string Render(USFMDocument input)
                 output.AppendLine($"<div class=\"{class_name}\">");
             }
 
+            // Will add a table with two columns where the second column will be blank
+            if(ConfigurationHTML.blankColumn)

```

Comment:
Can you try to make this in CSS?

Reply:
@rbnswartz Do you mean avoid using table?